HISTORY.rst

```diff
@@ -10,6 +10,8 @@ Unreleased
 
```

Comment:
- BREAKING: Change default `overwrite` value for ``lyrics.modules.update_translations`` to False